docs/section22/Rule22-2.md

```diff
@@ -0,0 +1,42 @@
+
+# CHW&CW - Rule 22-2  
+
+**Rule ID:** 22-2  
+**Rule Description:** When baseline building requires CHW loop, design CHW Loop return temperature is 56F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.8 Chilled-water design supply temperature (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13
+
```

Comment:
Would include same applicability checks as for 22-1 (that there is only one CHW plant)

Reply:
Added.

Reply:
Same applicability check comment